packages/calypso-css-custom-properties/bin/build-custom-properties-css.js

```diff
@@ -1,10 +1,14 @@
-const { dirname } = require( 'path' );
-const { mkdirSync, writeFileSync } = require( 'fs' ); // eslint-disable-line import/no-nodejs-modules
+/* eslint-disable import/no-nodejs-modules */
+const { dirname, join } = require( 'path' );
+const { existsSync, mkdirSync, writeFileSync } = require( 'fs' );
 const { renderSync } = require( 'node-sass' );
 
-const INPUT_FILE = 'src/custom-properties.scss';
-const OUTPUT_FILE = 'dist/custom-properties.css';
+const INPUT_FILE = join( __dirname, '..', 'src', 'custom-properties.scss' );
+const OUTPUT_FILE = join( __dirname, '..', 'dist', 'custom-properties.css' );
+
+if ( ! existsSync( dirname( OUTPUT_FILE ) ) ) {
+	mkdirSync( dirname( OUTPUT_FILE ), { recursive: true } );
+}
```

Comment:
I didn't add and `exitsSync` check since my reading (and testing) of the `recursive: true` arg was that it worked even if the path existed already :grimacing: (But I should've commented to that effect.)